src/utils/getUserProfile.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * @flow
+ */
+
+import { get } from 'immutable';
+import { LangUtils } from 'lattice-utils';
+
+import type { UserProfile } from '../types';
+
+const { isNonEmptyString } = LangUtils;
+
+export default function getUserProfile(user :any) :UserProfile {
+
+  const auth0UserProfile = get(user, 'profile', user);
+
+  let familyName = get(auth0UserProfile, 'family_name');
+  if (!isNonEmptyString(familyName)) {
+    familyName = '';
+  }
+
+  let givenName = get(auth0UserProfile, 'given_name');
+  if (!isNonEmptyString(givenName)) {
+    givenName = '';
+  }
+
+  let name = get(auth0UserProfile, 'name');
+  if (!isNonEmptyString(name)) {
+    name = '';
+  }
```

Comment:
Won't providing a fallback value (empty string) for the immutable `get` achieve the same goal in these three instances?

Reply:
Well... yea... I guess I'm just being extra defensive in the case where any of these fields are not a strings, which should not be possible, but 🤷‍♂️